services/device-registry-base/src/test/java/org/eclipse/hono/service/management/credentials/CredentialsTest.java

```diff
@@ -83,12 +86,14 @@ public void testEncodePasswordCredential() {
     @Test
     public void testEncodePskCredential() {
         final PskCredential credential = new PskCredential();
+                credential.setAuthId("psk-device");
+                credential.setSecrets(List.of(new PskSecret().setKey(new byte[] { 0x00, 0x01 })));
 
         final JsonObject json = JsonObject.mapFrom(credential);
+        System.out.println(json);
```

Comment:
Is `System.out.println` here by intention?

Reply:
Nice catch :+1: It was, but I forgot to remove it ...